apps/test/unit/storage/FirebaseStorageTest.js

```diff
@@ -160,6 +160,23 @@ describe('FirebaseStorage', () => {
           });
       }
     });
+    it('escapes periods in the key', done => {
+      FirebaseStorage.setKeyValue(
+        'foo.bar',
+        'baz',
+        () => verifyValue(),
+        err => console.log(err)
```

Comment:
sorry to be dense here, but is this accidental debugging code or does this console.log cause the test to fail somehow (how?) I know that console.warn can cause tests to fail in some situations but didn't think console.log did.

also, I see some similar uses of didWarn earlier in the file. not necessary to fix in this PR, but since it sounds like you were able to find and fix a problem here, it could be worth checking if the same fix is needed there.

Reply:
ok, here is i think where we might have a disconnect -- you mentioned in your other comment that the test passes when you escape the periods, and fails otherwise. this is great, and the most important aspect of the test. what I missed earlier, but now I see, is that only verifyValue() calls the done() handler. so if there is an error (i.e. the error handler is called, not the success handler), the test *will* fail, because done() will not be called. 

earlier in the file, there is a similar pattern, didWarn is present but doesn't do anything, but the done() handler ensures that the test will fail if the error handler is called. so I think you could make the same change there that you made here, and remove didWarn.